src/types/events/message-events.spec.ts

```diff
@@ -0,0 +1,158 @@
+// tslint:disable:no-implicit-dependencies
+import { assert } from 'chai';
+import { BotMessageEvent, MessageEvent } from './message-events';
+
+describe('message event types', () => {
+  it('should be compatible with bot_message payload', () => {
+    const payload: BotMessageEvent = {
+      type: 'message',
+      subtype: 'bot_message',
+      bot_id: '',
+      username: '',
+      icons: {},
+      channel: '',
+      text: '',
+      blocks: [],
+      thread_ts: '',
+      ts: '',
+      event_ts: '',
+      channel_type: 'channel',
+    };
+    assert.isNotEmpty(payload);
```

Comment:
Interesting. I've never seen types tested this way!

Reply:
We can do the same in the `types-tests` but having these is easier. Also, we can add other kinds of tests in the same place too.

Reply:
I defer to you on this if you think it best. Testing types is not something I'm well-versed in enough to have a strong opinion (yet!). 🙂